docs(kic): use generated CRDs reference (#4813)
Kong/docs.konghq.com: app/_includes/md/kubernetes-ingress-controller/udp_ingress_description.md

> It makes it possible to route traffic to your UDP services using Kong (e.g. DNS, Game Servers, etc.). 

Reviewer: ```suggestion
It makes it possible to route traffic to your UDP services using Kong (for example, DNS or Game Servers). 
```

> For each rule provided in the spec the Kong proxy environment must be updated to listen to UDP on that port as well.

Reviewer: ```suggestion
For each rule provided in the spec, the Kong proxy environment must be updated to listen to UDP on that port as well.
```
